CONTRIBUTING.md

```diff
@@ -45,16 +45,14 @@ querying network status and applying network states.
 
 ## Installation and Compilation:
```

Comment:
The order is reversed, first you have to compile and then you can optionally install. I would change the title to "Compile and install from source".